charts/rstudio-pm/values.yaml

```diff
@@ -147,6 +147,16 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
+serviceMonitor:
+  # -- If true, create a ServiceMonitor CRD for use with a Prometheus Operator
+  enabled: false
+  # -- Namespace to create the ServiceMonitor in (usually the same as the one in
+  # which the Operator is running). Defaults to the one used by the chart
+  namespace: null
```

Comment:
The pattern that we often use for this is `""`, because that keeps the type the same and allows `default` to still work

https://helm.sh/docs/chart_template_guide/function_list/#default

i.e. here:
https://github.com/rstudio/helm/blob/33f517f0c1b5015decfbf6a6424f182ff4e39ebe/charts/rstudio-connect/values.yaml#L179-L183